src/ansys/dpf/core/model.py

```diff
@@ -469,6 +469,27 @@ def _mesh_provider_cached(self):
             self._mesh_provider_cached_instance = self.mesh_provider
         return self._mesh_provider_cached_instance
 
+    @property
+    def mesh_info_provider(self):
+        """Mesh info provider operator.
+
+        This operator gives the information of a mesh from the result file. The underlying
+        operator symbol is the class:`ansys.dpf.core.operators.metadata.mesh_info_provider`
+        operator.
+
+        Returns
+        -------
+        mesh_info_provider : :class:`ansys.dpf.core.operators.metadata.mesh_info_provider`
+            Mesh info provider operator.
+
+        """
+        mesh_info_provider = Operator("MeshInfoProvider", server=self._server)
```

Comment:
Did we finally support `"MeshInfoProvider"` and `"mesh_info_provider"` or only the latter?